examples/pipelines/simple_pipeline/components/load_from_hub/src/main.py

```diff
@@ -0,0 +1,74 @@
+"""
+This component loads a seed dataset from the hub.
+"""
+import io
+import logging
+
+from datasets import load_dataset
+
+import dask.dataframe as dd
+import dask.array as da
+import pandas as pd
+import numpy as np
+from PIL import Image
+
+from express.dataset import FondantComponent
+from express.logger import configure_logging
+
+configure_logging()
+logger = logging.getLogger(__name__)
+
+
+def extract_width(image_bytes):
+    # Decode image bytes to PIL Image object
+    pil_image = Image.open(io.BytesIO(image_bytes["bytes"]))
+    width = pil_image.size[0]
+    
+    return width
+
+def extract_height(image_bytes):
+    # Decode image bytes to PIL Image object
+    pil_image = Image.open(io.BytesIO(image_bytes["bytes"]))
+    height = pil_image.size[1]
+    
+    return height
+
+
+class LoadFromHubComponent(FondantComponent):
+    type = "load"
+
+    @classmethod
+    def load(cls, args):
+        """
+        Args:
+            args: additional arguments passed to the component
+        
+        Returns:
+            Dataset: HF dataset
+        """
+        # 1) Load data, read as Dask dataframe
+        logger.info("Loading dataset from the hub...")
+        dataset = load_dataset(args.dataset_name, split="train")
```

Comment:
hmm Not really a big fan or this approach (loading, writing and reading again with another framework) but I guess it's fine for now. Maybe create a ticket for it?

Reply:
What's the alternative? Starting from a dataset in a bucket?

Reply:
that or writing/git clone directly to the bucket and then reading the data from there. That way we avoid reading the data in memory. Although I'm not sure how to achieve this. Can be tackled later